website/static/js/pages/wiki-edit-page.js

```diff
@@ -1,16 +1,153 @@
+    compareVisible: comparable,
+    canEdit: ctx.canEdit,
+    viewVersion: 'current',
+    compareVersion: 'current',
+    urls: ctx.urls,
+    metadata: ctx.metadata
+};
+
+var wikiPage = new WikiPage('#wikiPageContext', wikiPageOptions);
+
+
+// Edit wiki page name
+if (ctx.canEditPageName) {
+    // Initialize editable wiki page name
+    var $pageName = $('#pageName');
+    $.fn.editable.defaults.mode = 'inline';
+    $pageName.editable({
+        type: 'text',
+        send: 'always',
+        url: ctx.urls.rename,
+        ajaxOptions: {
+            type: 'put',
+            contentType: 'application/json',
+            dataType: 'json'
+        },
+        validate: function(value) {
+            if($.trim(value) === ''){
+                return 'The wiki page name cannot be empty.';
+            } else if(value.length > 100){
+                return 'The wiki page name cannot be more than 100 characters.';
+            }
+        },
+        params: function(params) {
+            return JSON.stringify(params);
+        },
+        success: function(response, value) {
+            window.location.href = ctx.urls.base + encodeURIComponent(value) + '/';
+        },
+        error: function(response) {
+            var msg = response.responseJSON.message_long;
+            if (msg) {
+                return msg;
+            } else {
+                // Log unexpected error with Raven
+                Raven.captureMessage('Error in renaming wiki', {
+                    url: ctx.urls.rename,
+                    responseText: response.responseText,
+                    statusText: response.statusText
+                });
+                return 'An unexpected error occurred. Please try again.';
+            }
+        }
+    });
+}
+
+// Apply panels
+$(document).ready(function () {
+    $('*[data-osf-panel]').osfPanel({
+        buttonElement : '.switch',
+        onSize : 'md',
+        'onclick' : function (title, thisbtn, event ) {
+            // this = all the column elements; an array
+            // title = Text of the button
+            // thisbtn = $(this);
+            // thisbtn.hasClass('btn-primary')
 
-ShareJSDoc('.wiki', url, metadata);
\ No newline at end of file
+            $('body').trigger('togglePanel', [
+                title.toLowerCase(),
+                thisbtn.hasClass('btn-primary')
+            ]);
+
+            if (typeof editor !== 'undefined') { ace.edit(editor).resize(); }
+
+        }
+    });
+    $('.openNewWiki').click(function () {
+        $('#newWiki').modal('show');
+    });
+    $('.openDeleteWiki').click(function () {
+        $('#deleteWiki').modal('show');
+    });
+    var panelToggle = $('.panel-toggle'),
+        panelExpand = $('.panel-expand');
+    $('.panel-collapse').on('click', function () {
+        var el = $(this).closest('.panel-toggle');
+        el.children('.wiki-panel.hidden-xs').hide();
+        panelToggle.removeClass('col-sm-3').addClass('col-sm-1');
```

Comment:
I suspect this is because of the collapsible menu, yes? That interrupted the cleanliness of the column design.